src/background.js

```diff
@@ -0,0 +1,15 @@
+//TODO Change sample code to acceot scripting and option to apply on any page
+chrome.runtime.onInstalled.addListener(function() {
+    chrome.storage.sync.set({color: '#3aa757'}, function() {
```

Comment:
Here I'd tell you to use the Arrow Functions: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/